commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/offloaded/OffloadedDrive.scala

```diff
@@ -2,20 +2,20 @@ package com.lynbrookrobotics.potassium.commons.drivetrain.offloaded
 
 import com.lynbrookrobotics.potassium.Signal
 import com.lynbrookrobotics.potassium.commons.drivetrain.twoSided.{TwoSided, TwoSidedDrive}
-import com.lynbrookrobotics.potassium.commons.drivetrain.{ForwardPositionGains, ForwardVelocityGains}
-import com.lynbrookrobotics.potassium.control.OffloadedSignal.{EscPositionGains, EscVelocityGains}
-import com.lynbrookrobotics.potassium.control._
+import com.lynbrookrobotics.potassium.control.offload.OffloadedSignal.{OpenLoop, PositionControl, VelocityControl}
+import com.lynbrookrobotics.potassium.control.offload.{EscConfig, OffloadedSignal}
 import com.lynbrookrobotics.potassium.streams.Stream
-import com.lynbrookrobotics.potassium.units.GenericIntegral
-import squants.motion.Acceleration
 import squants.space.Length
-import squants.time.{Milliseconds, Seconds}
 import squants.{Dimensionless, Velocity}
 
 abstract class OffloadedDrive extends TwoSidedDrive {
   override type DriveSignal = TwoSided[OffloadedSignal]
   override type Properties <: OffloadedProperties
 
+  implicit val escConfig: EscConfig[Length]
```

Comment:
One thing I was thinking. Would it be possible to completely remove the translation from here and make it the responsibility of `applySignal`?

Reply:
What purpose would `OffloadedDrive` serve then?

Reply:
Didn't we decide that `applySignal` should be as low-level as possible?

Reply:
Well offloaded would still handle replacing all controllers with offloaded ones. I'm just not super comfortable with this half abstraction where normally only components care about what speed controller is used but here we're leaking that into the drive module. Just something to think about.